### ISSUE TITLE
Support `def` for checking defined symbols

### ISSUE THREAD

**Issue by vladikcomper** [Clownacy/clownassembler#5]
This is merely a feature request, not a bug report, but seeing how clownassembler already supports expressions in `if`s I thought it shouldn't take too long to implement.

Basically, one of the most useful features for any project that supports build options (or multiple builds) in original ASM68K is to come up with "compile flags" defined in assembler arguments and use `if def(...)` in the source files to check for them, e.g.:

```m68k
    if def(_DEBUG_)  ; is it debug build?
        ; Some extra logic here
    endc

    if def(_DEBUG_)=0 ; NOT a debug build?

    endc
```

As noted, "compile flags" can be passed to assembler arguments, but are otherwise completely optional:

```sh
    # DEBUG build (defines a "compile flag")
    $(ASM68K) /e _DEBUG_ main.asm, main.bin

    # RELEASE build (nothing)
    $(ASM68K) main.asm, main.bin
```

Basically, I've been looking into compiling at least least blobs for my [MD Debugger](https://github.com/vladikcomper/md-modules) using clownassembler and unlike actual debugger macros, they are relatively straightforward and I found out your assembler supports almost everything by now except for `def` function and `/e` option (it actually has two syntaxes: `/e SYMBOL` and `/e SYMBOL=value`)

**Comment by Clownacy** [comment]
I've added support for these to both frontends. I've made sure to test a handful of edge-cases, but there may be some that I missed. so feel free to let me know if you encounter any problems.
